internal/cli/atlas/streams/connection/connection.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 MongoDB Inc
```

Comment:
Could you add e2e tests to make sure that the commands work as expected? see https://github.com/mongodb/mongodb-atlas-cli/tree/master/test